client/my-sites/site-settings/disconnect-site/down-flow.jsx

```diff
@@ -0,0 +1,118 @@
+import Gridicon from 'components/gridicon';
+import Main from 'components/main';
+import NavigationLink from 'components/wizard/navigation-link';
+import { addQueryArgs } from 'lib/url';
+import { localizeUrl } from 'lib/i18n-utils';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { useTranslate } from 'i18n-calypso';
+
+function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } ) {
+	const translate = useTranslate();
+
+	return (
+		<Main className="disconnect-site__down-flow">
+			<DocumentHead title={ translate( 'Site Settings' ) } />
+			<FormattedHeader
+				headerText={ translate(
+					'Your site, {{strong}}%(siteName)s{{/strong}}, cannot be accessed',
+					{
+						args: { siteName: site.name },
+						components: { strong: <strong /> },
+					}
+				) }
+				subHeaderText={ translate(
+					'Jetpack wasn’t able to connect to {{strong}}%(siteSlug)s{{/strong}}.{{br/}}Let’s figure out why — there are a few things to try.',
+					{
+						args: { siteSlug: site.slug },
+						components: {
+							strong: <strong />,
+							br: <br />,
+						},
+					}
+				) }
+			/>
+			<div className="disconnect-site__actions">
+				<CompactCard
+					onClick={ () => tracks( 'calypso_jetpack_site_indicator_disconnect_confirm' ) }
+					href={ site.URL }
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					<Gridicon
+						className="disconnect-site__action-icon disconnect-site__confirm-icon"
+						icon="globe"
+					/>
+					<FormattedHeader
+						isSecondary
+						align="left"
+						headerText={ translate( 'Confirm that your site loads.' ) }
+						subHeaderText={ translate(
+							'Visit your site to make sure it loads properly. If there’s an issue, fix your site before worrying about Jetpack! That may resolve this error.'
+						) }
+					/>
+				</CompactCard>
+				<CompactCard
+					onClick={ () => tracks( 'calypso_jetpack_site_indicator_disconnect_troubleshoot' ) }
+					href={ localizeUrl(
+						'https://jetpack.com/support/getting-started-with-jetpack/fixing-jetpack-connection-issues/'
+					) }
+				>
+					<Gridicon
+						className="disconnect-site__action-icon disconnect-site__troubleshoot-icon"
+						icon="plans"
+					/>
+					<FormattedHeader
+						isSecondary
+						align="left"
+						headerText={ translate( 'Troubleshoot Jetpack.' ) }
+						subHeaderText={ translate(
+							'If your site is working but you’re still seeing this error, let’s troubleshoot your Jetpack connection.'
+						) }
+					/>
+				</CompactCard>
+				<CompactCard
+					onClick={ () => tracks( 'calypso_jetpack_site_indicator_disconnect_disconnect' ) }
+					href={ addQueryArgs( { type: 'down' }, confirmHref ) }
+				>
+					<Gridicon
+						className="disconnect-site__action-icon disconnect-site__disconnect-icon"
+						icon="cross-circle"
+					/>
+					<FormattedHeader
+						isSecondary
+						align="left"
+						headerText={ translate( 'Disconnect Jetpack.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **43** times:
`translate( 'Disconnect Jetpack' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-cannot-be-accessed-flow)